UnitTesting-master/src/test/java/seleniumMail/Tests/YandexMailTest.java

```diff
@@ -0,0 +1,41 @@
+package seleniumMail.Tests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import seleniumMail.Helpers.PropertiesUtil;
+import seleniumMail.Helpers.WebDriverSingleton;
+import seleniumMail.Pages.InboxPage;
+import seleniumMail.Pages.LoginPage;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class YandexMailTest {
+
+    private static final String USERNAME = "user1";
+    private static final String PASSWORD = "password1";
+    private static String user = PropertiesUtil.get(USERNAME);
+    private static String password = PropertiesUtil.get(PASSWORD);
+
+    @Test
+    public void logInTest() {
+        LoginPage loginPage = new LoginPage();
+        loginPage.logIn(user, password);
+        InboxPage inboxPage = new InboxPage();
```

Comment:
In 99% cases you will create LoginPage via constructor in tests only, another pages will be result of actions. Please let me know, if you need additional explanation from me
`InboxPage inboxPage = loginPage.logIn(user, password);`